cli/src/commands/mod.rs

```diff
@@ -2141,7 +2144,24 @@ fn cmd_commit(ui: &mut Ui, command: &CommandHelper, args: &CommitArgs) -> Result
     let matcher = workspace_command.matcher_from_values(&args.paths)?;
     let mut tx = workspace_command.start_transaction(&format!("commit {}", commit.id().hex()));
     let base_tree = merge_commit_trees(tx.repo(), &commit.parents())?;
-    let tree_id = tx.select_diff(ui, &base_tree, &commit.tree()?, matcher.as_ref(), "", false)?;
+    let instructions = format!(
+        "\
+You are making a partial commit from the changes in: {}
+
+The diff initially shows all changes. Adjust the right side until it shows the
+contents you want for the new commit. The remainder will stay in the working
+copy.
```

Comment:
Since the new working-copy commit has a different change id, I wouldn't say "stay". Maybe "The remainder will be included in the new working-copy commit."?

Reply:
I made this change, and also changed "new commit" to "first commit" instead for consistency with our other `jj commit` messaging.